sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -155,8 +155,6 @@ public static DataSourceConfiguration create(
         String driverClassName, String url, String username, String password) {
       checkNotNull(driverClassName, "driverClassName");
       checkNotNull(url, "url");
```

Comment:
- Yes, username can be null. Only driver class name and url can't be null.
- +1